controller/src/main/java/org/jboss/as/controller/CapabilityReferenceRecorder.java

```diff
@@ -156,8 +165,9 @@ private void processCapabilityRequirement(OperationContext context, String attri
          *
          * @param currentAddress the address of the resource currently being processed. Will not be {@code null}
          * @return the dynamic portion of the dependenty capability name. Cannot be {@code null}
+         * @deprecated No longer used.
          */
-        String getDynamicDependentName(PathAddress currentAddress) {
+        @Deprecated String getDynamicDependentName(PathAddress currentAddress) {
```

Comment:
this is a (package) method. If it is no longer used, we can delete it instead of deprecating it.

Reply:
That's true.  I'll remove it.